docs/contributing.rst

```diff
@@ -63,6 +63,32 @@ To build the documentation, invoke Sphinx from the ``docs`` directory::
     $ cd docs
     $ make html
 
+Extension Packages
+------------------
+
+Pint naturally integrates with other libraries in the scientific Python ecosystem, and
+a small number of
+`extension/compatibility packages<numpy.html#Compatibility-Packages>`_ have arisen to
+aid in compatibility between certain packages. Pint's rule of thumb for integration
+features that work best as an extension pacakage verses direct inclusion in Pint is:
```

Comment:
typo: versus

Reply:
Thank you catching that! Update has been made.